crates/core/c8y_api/src/smartrest/operations.rs

```diff
@@ -260,6 +289,21 @@ pub fn get_operations(dir: impl AsRef<Path>) -> Result<Operations, OperationsErr
                 Err(err) => return Err(OperationsError::FromIo(err)),
             };
 
+            // replace bridge prefix template with mapper prefix (if necessary)
+            if let Some(ref mut exec) = details.exec {
+                if let Some(index) = exec
+                    .topic
+                    .as_ref()
+                    .and_then(|topic| topic.find(BRIDGE_PREFIX_TEMPLATE))
+                {
+                    // safe to unwrap as we checked that topic exists when looking for bridge prefix template
+                    exec.topic
+                        .as_mut()
+                        .unwrap()
+                        .replace_range(index..index + BRIDGE_PREFIX_TEMPLATE.len(), prefix);
+                }
+            }
+
```

Comment:
The workflow code that does the variable expansion can be found here: https://github.com/thin-edge/thin-edge.io/blob/be82810edc0ba5d4d68712617d627aa3018d30d4/crates/core/tedge_api/src/workflow/state.rs#L294

Reply:
Indeed, it would be better to use a similar template expansion. However, the workflow implementation cannot directly be reused here. I will take the time to see how this code can be shared.

Reply:
The following commit will help to fix that: https://github.com/thin-edge/thin-edge.io/pull/3204/commits/0bd5274eff1122ed38c3ca5c8472af470472f45a

By implementing the `Record` trait for the bridge config (a single method returning a Json value for a given), this config can then be used to inject value into a string or a script.